src/main/java/no/ecc/vectortile/VectorTileEncoder.java

```diff
@@ -169,11 +174,23 @@ public void addFeature(String layerName, Map<String, ?> attributes, Geometry geo
 
         layer.features.add(feature);
     }
-    
+
+    /**
+     * A short circuit clip to the tile extent (tile boundary + buffer) for points to improve performance.
+     * @see https://github.com/ElectronicChartCentre/java-vector-tile/issues/13
+     */
+    protected Geometry clipToExtent(Point geom) {
```

Comment:
If this method is not to change the Point, then perhaps returning a boolean would be better? That way, you do not need to return a empty geometry?